spec/lib/services/api/metric_rollups_service_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe Api::MetricRollupsService do
+  before do
+    allow(Settings.api).to receive(:metrics_default_limit).and_return(1000)
```

Comment:
Is this stub important to the actual values returned, or more of a sane default? If the latter, I wonder if it  might be better placed as an actual configuration in the test environment, in `config/settings/test.yml`?